File: VS Projects/PyPassGen/dict.cs
namespace dict {
    
    using @string;
    
    using System.Linq;
    
    using System.Collections.Generic;
    
    public static class Module {
        
        public static object lowercase_letter_list = @string.ascii_lowercase.ToList();
        
        public static object symbol_list = new List<object> {
            "|",
            "Ä",
            "€",
            "÷",
            "×",
            "ä",
            "đ",
            "[",
            "]",
            "ł",
            "Ł",
            "$",
            "ß",
            "¤",
            "<",
            ">",
            "#",
            "&",
            "@",
            "{",
            "}*",
            "/",
            "+",
            "-",
            ".",
            ","
        };
        
        public static object uppercase_letter_list = @string.ascii_uppercase.ToList();
    }
}

File: Python/Terminal/RandomPassGen/dict.py
import string
import random

letter_list = list(string.ascii_letters)

lowercase_letter_list = list(string.ascii_lowercase)
uppercase_letter_list = list(string.ascii_uppercase)

digit_list = list(string.digits)

symbol_list = [
'§', '%', '=', '/', '(', ')','|', 'Ä', '€', '÷', '×', 'ä', 'đ', '[', ']', 'ł', 'Ł', '$',
'ß', '¤', '<', '>', '#', '&', '@', '{', '}', ';', ':', ',', '.', '-', '_', '˘', '~', '^']

random.shuffle(lowercase_letter_list)
random.shuffle(uppercase_letter_list)
random.shuffle(symbol_list)
random.shuffle(digit_list)
random.shuffle(letter_list)